zb-db/src/main/java/io/camunda/zeebe/db/ColumnFamily.java

```diff
@@ -117,6 +117,26 @@
    */
   void whileEqualPrefix(DbKey keyPrefix, KeyValuePairVisitor<KeyType, ValueType> visitor);
 
+  /**
+   * Visits the key-value pairs, which are stored in the column family and which have the same
+   * common prefix. The ordering depends on the key. The visitor can indicate via the return value,
+   * whether the iteration should continue or * not. This means if the visitor returns false the
+   * iteration will stop.
+   *
+   * <p>The given {@code startAtKey} indicates where the iteration should start. If the key exists,
+   * the first key-value-pair will contain the equal key as {@code startAtKey}. If the key doesn't
+   * exist it will start after.
+   *
+   * <p>Similar to {@link #whileEqualPrefix(DbKey, BiConsumer) and {@link
+   * #whileTrue(KeyValuePairVisitor)}}.
+   *
+   * @param keyPrefix the prefix which should have the keys in common
+   * @param startAtKey indicates on which key the iteration should start
+   * @param visitor the visitor which visits the key-value pairs
+   */
+  void whileEqualPrefix(
```

Comment:
:question: Do you think this will cause a failure in `zeebe-process-test`? Do we need a ticket for that there? 

I remember that @korthout had to implement some similar changes when ZDP extended this API a month (or so) ago.

Reply:
Yes, I have an issue for this in the epic already. Good catch!